### PR TITLE
Update V&V_request.md

### PR DESCRIPTION
**Pull request by abbylindsay**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/V&V_request.md [hunk 1]
```diff
@@ -69,6 +69,8 @@ Application:
 - [ ] Device enrolled in Hexnode
 - [ ] Equipment set up for V&V execution
 - [ ] Parse account created in correct server
+- [ ] Verify if V&V account should be an internal or external parse account
+- [ ] Set expiration date to parse account
```
Comment (barchard) [inline .github/ISSUE_TEMPLATE/V&V_request.md:73]: Couple things:

1. `Parse` is capitalized
2. Upon completion of the V&V, set the expiration date of the Parse account used during the V&V to the current date and time.

**Comment by ana-enriquez** [comment]
Instead of "Set expiration date to parse account" I would write it as "Enter 'study end date' for Parse account in CLI tools" so its a bit more clear.

**Comment by abbylindsay** [comment]
@MarcDerhammer how should the clear previous account item be expressed in this checklist. is has to be an engineering task?

**Comment by abbylindsay** [comment]
i capitalized Parse and changed the expiration item to ana's suggestion. there might need to be an item added to engineering?

**Review by barchard — APPROVED** [review]

**Pull request merged**